docs/Tutorials/EventsAndTriggers.md

```diff
@@ -0,0 +1,72 @@
+\cond NEVER
+Distributed under the MIT License.
+See LICENSE.txt for details.
+\endcond
+
+# Events and triggers {#tutorial_events_and_triggers}
+
+The "events and triggers" infrastructure (and the related "events and
+dense triggers") provide some control of SpECTRE executable execution
+from the input file contents.  It does not generally have major
+effects on the flow of the algorithm, but is primarily used for
+observations and minor adjustments.
+
+### What is an event?
```

Comment:
I'm personally not a fan of the "question and answer" type of documentation. Also I don't think it's used anywhere else in spectre (if it is used, it's sparse). For consistency I think we should keep it as
- Event
- Trigger
- Dense Trigger
- Input File Syntax